back-nest/src/cart/cart.service.ts

```diff
@@ -0,0 +1,57 @@
+import { Injectable, Logger } from '@nestjs/common';
+
+import { PrismaService } from '../../prisma/prisma.service';
+import { UpdateCartParams } from './model/updateCartParams.model';
+
+@Injectable()
+export class CartService {
+  private readonly logger = new Logger(CartService.name);
+
+  constructor(private prisma: PrismaService) {}
+
+  async getUserCart(user_id: number) {
```

Comment:
Use camelcase naming style in your code. For database, don't make any changes. Apply this for every other snake_case names